src/Microsoft.Extensions.Options.ConfigurationExtensions/ConfigureFromConfigurationOptions.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.Extensions.Options
 {
-    public class ConfigureFromConfigurationOptions<TOptions> : ConfigureOptions<TOptions>
+    public class ConfigureFromConfigurationOptions<TOptions> : ConfigureOptions<TOptions>, IOptionsConfiguration<TOptions>
```

Comment:
Alternation seems mathematically correct and a bit less obscure that mutator or mutation, at least IMO :smile:

Yeah, I like setup too.